### PR TITLE
Note - VIC 1.5.2 & later don't support Flex client

### LINKED ISSUE #2429: Document that Flex client plugin is not shipped with 1.5.2
Per https://github.com/vmware/vic-product/pull/2330. Confirmed by @wjun (and seen by myself in the Flex client) that from 1.5.2 we no longer ship the Flex client plugin.

### PR DESCRIPTION
**Pull request by vidya-v**
Hi @stuclem ,

I added a note to say that VIC 1.5.2 and later versions don't include the Flex client.
Do I have to add the note in all topics that mention the Flex client?

Fixes #2429 

Thanks,
Vidya

### CODE REVIEW

**Comment by stuclem** [comment]
Hi @vidya-v it might also be worth mentioning in https://vmware.github.io/vic-product/assets/files/html/latest/vic_vsphere_admin/vch_portlet_ui.html and https://vmware.github.io/vic-product/assets/files/html/latest/vic_vsphere_admin/deploy_vic_appliance.html too.

**Comment by vidya-v** [comment]
Thanks, @stuclem . I have added the note to the two topics.

File: docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md [hunk 1]
```diff
@@ -26,7 +26,9 @@ If you need to deploy multiple appliances, you can use the initialization API to
...
 - If you use vCenter Server 6.7 update 1 or later, you can use the HTML5 vSphere Client to deploy the appliance. If you use an older version of vCenter Server, you must use the Flex-based vSphere Web Client to deploy the appliance. You cannot deploy OVA files from versions of the HTML5 vSphere Client that pre-date vCenter Server 6.7 update 1. 
 
-    **NOTE**: Versions of the HTML5 client that pre-date 6.7 update 1 do not prevent you from deploying OVA files and deployment appears to succeed. However, the resulting appliance [does not function correctly due to an issue with the HTML5 client](ts_reg_doesnt_start.md). This issue is fixed in version 6.7 update 1 of the vSphere Client. 
+    **NOTE**: Versions of the HTML5 client that pre-date 6.7 update 1 do not prevent you from deploying OVA files and deployment appears to succeed. However, the resulting appliance [does not function correctly due to an issue with the HTML5 client](ts_reg_doesnt_start.md). This issue is fixed in version 6.7 update 1 of the vSphere Client.
+    
+    vSphere Integrated Containers 1.5.2 and later versions do not include the Flex-based vSphere Web Client.
```
Comment (stuclem) [inline docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: I think that this note would be better placed under the bullet that mentions "A basic informational plug-in for the Flex-based vSphere Web Client" above

Comment (vidya-v) [reply docs/user_doc/vic_vsphere_admin/deploy_vic_appliance.md]: Done.

**Comment by vidya-v** [comment]
> LGTM!

Thank you, @stuclem

**Pull request merged**